mp/src/game/client/momentum/ui/menus/hud_menu_static.h

```diff
@@ -47,6 +48,8 @@ class CHudMenuStatic : public CHudElement, public vgui::Panel
   private:
     void (*SelectFunc)(int);
     void (*CloseFunc)();
```

Comment:
Is there any reason to be a close func if the subclasses were deleted?

Reply:
There isn't no, unless something else wants to use it in the future. Safe to remove?

Reply:
Yep should be safe to remove.